docker/Dockerfile

```
@@ -195,6 +195,12 @@ ENV USER=${USER}
 ARG HOME
 ENV HOME=${HOME}
 
+# Install curl for healtchecks
+RUN apt-get -qq update && \
+    apt-get -qq install -y --no-install-recommends \
```

Is the flag meant to be `--quiet`? I don't see a `-qq` flag in the `apt-get` reference. Using the long name of the flag might be nice.



```suggestion
RUN apt-get -q update && \
    apt-get -q install -y --no-install-recommends \
```

---

In their man page they refer to it this way:

> More q's will produce more quiet up to a maximum of two. You can also use -q=# to set the quiet level, overriding the configuration file. Note that quiet level 2 implies -y

We want to avoid any kind of dialogues, and if they appear by any chance, then accept them.

But yes, it's redundant because we already have `-y`